change/@fluentui-react-monaco-editor-2021-01-22-19-28-04-hotell-build-system-ts-migration-1.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "none",
+  "comment": "chore: switch to single version policy with typescript",
+  "packageName": "@fluentui/react-monaco-editor",
+  "email": "martinhochel@microsoft.com",
+  "dependentChangeType": "none",
```

This change does affect the published package
```suggestion
  "type": "patch",
  "comment": "Remove typescript dependency",
  "packageName": "@fluentui/react-monaco-editor",
  "email": "martinhochel@microsoft.com",
  "dependentChangeType": "patch",
```

---

so does it change it or not ? (as you mentioned TS is shipped with monaco)

---

Yes, the package changes because its `dependencies` (not `devDependencies`) changed, even though the change was to remove something.